StreamChatUITestsApp/StreamChat/StreamChatWrapper.swift

```diff
@@ -34,8 +35,48 @@ final class StreamChatWrapper {
         // UI
         let query = ChannelListQuery(filter: .containMembers(userIds: [userCredentials?.id ?? ""]))
         let controller = ChatClient.shared.channelListController(query: query)
-        let channelList = ChatChannelListVC.make(with: controller)
+        let channelList = ChannelList.make(with: controller)
         return channelList
     }
 
 }
+
+final class ChannelList: ChatChannelListVC, ChatConnectionControllerDelegate {
```

Comment:
@alteral please move this class to separate file.

Reply:
👍